### DISCUSSION TITLE
"Done-File" doesnt works

### DISCUSSION THREAD

**Discussion by Enricone27** [vzhd1701/enex2notion#19] [Q&A]
I'm uploading something with a donefile. But when I continued it started from the beginning.
I simply executed the same command as bevore.

**Comment by vzhd1701** [comment]
Can you show me example of your command?

**Reply by Enricone27** [reply]
enex2notion --token 5...8 --done-file done.txt Schule\ Math\ Gym2.enex

**Reply by vzhd1701** [reply]
Did it write anything into done.txt?

**Reply by Enricone27** [reply]
jap, there are 41 hashes, the first two and the last two hashes should be the same? They are different...
The first time it uploaded 39 and one unfinished and the second time 2 and one unfinished

**Reply by vzhd1701** [reply]
That's strange. All hashes should be different.
So the second time it didn't upload 39 that were previously uploaded, but some notes got reuploaded?

**Reply by Enricone27** [reply]
No, as I saw that it began to reupload the same first notes I canceled the task with Ctrl+C. So now I have 40 notes + 1 unfinished note + 2 notes + 1 unfinished note. And the first two notes of the 40 notes are the same as the other 2.

**Reply by vzhd1701** [reply]
Could you tell me what is your OS and how do you run the tool (portable binary from release page, pip, poetry)?

**Reply by Enricone27** [reply]
I run Linux Mint 20.3 with pipx
